client/jetpack-cloud/sections/pricing/style.scss

```diff
@@ -75,22 +75,6 @@
 		}
 	}
 
-	.plan-price {
```

Comment:
These early rules impact all pricing page iterations. Since the iterations we'd like to keep overwrite these styles, we can safely remove this.